neurom/apps/cli.py

```diff
@@ -4,18 +4,20 @@
 import click
 import matplotlib.pyplot as plt
 
+import neurom as nm
+from neurom.apps import morph_stats, morph_check
 from neurom import load_neuron
 from neurom.view.plotly import draw as plotly_draw
 from neurom.viewer import draw as pyplot_draw
 
-logging.basicConfig()
-logger = logging.getLogger('morph_tool')
-logger.setLevel(logging.INFO)
-
 
 @click.group()
-def cli():
+@click.option('-v', '--verbose', count=True, default=0,
+              help='-v for WARNING, -vv for INFO, -vvv for DEBUG')
+def cli(verbose):
     """The CLI entry point."""
+    level = (logging.WARNING, logging.INFO, logging.DEBUG, logging.DEBUG)[min(verbose, 2)]
```

Comment:
one extra debug

Reply:
one scotch, one bourbon, one debug!

Reply:
one debug, 2 bourbons, one whiskey, one ... , .... zzzzzzz

Reply:
guys, I can't stay focused with all this noise!

Reply:
have a drink then ...